tools/generators/dependency-mismatch/index.spec.ts

```diff
@@ -159,36 +170,74 @@ describe('dependency-mismatch generator', () => {
         [`${workspaceNpmScope}/tslib`]: '^2.1.1',
       },
       devDependencies: {},
+      peerDependencies: {},
     });
 
+    await generator(appTree);
+
     const packageJson: PackageJson = await readTargetPackageJson();
     expect(packageJson.dependencies).toMatchInlineSnapshot(`
       Object {
         "proj/tslib": "^2.1.1",
       }
     `);
   });
+
+  it('should ignore northstar packages', async () => {
```

Comment:
this is test is bit confusing. what is it showcasing ? can we make it better to be more explicit ?

Reply:
is `Generator should not run on northstar packages` more explicit ?